src/components/Icon/Icon.stories.mdx

```diff
@@ -29,111 +30,11 @@ All the icons available for use.
-        <PresentComponent name={'expenses'}>
-            <Icon name={'expenses'} />
-        </PresentComponent>
-        <PresentComponent name={'financialChart'}>
-            <Icon name={'financialChart'} />
-        </PresentComponent>
-        <PresentComponent name={'globalItem'}>
-            <Icon name={'globalItem'} />
-        </PresentComponent>
-        <PresentComponent name={'income'}>
-            <Icon name={'income'} />
-        </PresentComponent>
-        <PresentComponent name={'linked'}>
-            <Icon name={'linked'} />
-        </PresentComponent>
-        <PresentComponent name={'musicNote'}>
-            <Icon name={'musicNote'} />
-        </PresentComponent>
-        <PresentComponent name={'percentage'}>
-            <Icon name={'percentage'} />
-        </PresentComponent>
-        <PresentComponent name={'restore'}>
-            <Icon name={'restore'} />
-        </PresentComponent>
-        <PresentComponent name={'searchMusic'}>
-            <Icon name={'searchMusic'} />
-        </PresentComponent>
-        <PresentComponent name={'searchThunder'}>
-            <Icon name={'searchThunder'} />
-        </PresentComponent>
-        <PresentComponent name={'stamp'}>
-            <Icon name={'stamp'} />
-        </PresentComponent>
-        <PresentComponent name={'thunder'}>
-            <Icon name={'thunder'} />
-        </PresentComponent>
-        <PresentComponent name={'timeOut'}>
-            <Icon name={'timeOut'} />
-        </PresentComponent>
-        <PresentComponent name={'training'}>
-            <Icon name={'training'} />
-        </PresentComponent>
-        <PresentComponent name={'trophy'}>
-            <Icon name={'trophy'} />
-        </PresentComponent>
-        <PresentComponent name={'user'}>
-            <Icon name={'user'} />
-        </PresentComponent>
-        <PresentComponent name={'youtube'}>
-            <Icon name={'youtube'} />
-        </PresentComponent>
-        <PresentComponent name={'ticTacToeArrow'}>
-            <Icon name={'ticTacToeArrow'} />
-        </PresentComponent>
-        <PresentComponent name={'hourglass'}>
-            <Icon name={'hourglass'} />
-        </PresentComponent>
-        <PresentComponent name={'checkedList'}>
-            <Icon name={'checkedList'} />
-        </PresentComponent>
-        <PresentComponent name={'HourglassIcon'}>
-            <Icon name={'HourglassIcon'} />
-        </PresentComponent>
-        <PresentComponent name={'TicTacToeArrowIcon'}>
-            <Icon name={'TicTacToeArrowIcon'} />
-        </PresentComponent>
-        <PresentComponent name={'DotsVerticalIcon'}>
-            <Icon name={'DotsVerticalIcon'} />
-        </PresentComponent>
-        <PresentComponent name={'calendarEmpty'}>
-            <Icon name={'calendarEmpty'} />
-        </PresentComponent>
-        <PresentComponent name={'keyword'}>
-            <Icon name={'keyword'} />
+      {Object.keys(iconSelector).map((iconName) => (
```

Comment:
nice! This resolves an issue with the undefined icons rendered in mdx docs. 

The above solution replaces the names "TicTacToeArrowIcon, DotsVerticalIcon and HourglassIcon" with the iconSelector's key names ( ticTacToeArrow, dotsVertical, hourglass)

Reply:
Indeed!
I noticed it was giving an error after the addition of the 3 icons you mentioned above! 💣